### PR TITLE
Add new serializer config path

### PR DESCRIPTION
**Pull request by yceruto**
See https://github.com/symfony/symfony/pull/24833

### CODE REVIEW

File: serializer.rst [hunk 1]
```diff
@@ -22,7 +22,7 @@ it in your configuration:
 
     .. code-block:: yaml
 
-        # config/packages/framework.yaml
+        # config/packages/serializer.yaml
```
Comment (xabbuh) [inline serializer.rst]: this should probably not be changed

Comment (yceruto) [reply serializer.rst]: Indeed, thanks.

**Review by xabbuh — APPROVED** [review]

File: serializer.rst [hunk 2]
```diff
@@ -196,10 +196,10 @@ In addition to the ``@Groups`` annotation, the Serializer component also
 supports Yaml or XML files. These files are automatically loaded when being
 stored in one of the following locations:
 
-* The ``serialization.yml`` or ``serialization.xml`` file in
+* The ``serialization.yaml`` or ``serialization.xml`` file in
   the ``Resources/config/`` directory of a bundle;
-* All ``*.yml`` and ``*.xml`` files in the ``Resources/config/serialization/``
-  directory of a bundle.
+* All ``*.yaml`` and ``*.xml`` files in the ``Resources/config/serialization/``
+  directory of a bundle and in the ``config/serializer/`` directory of your project.
```
Comment (weaverryan) [inline serializer.rst]: Should we mention the `config/serializer/` directory if your project first *before* talking about `Resources/config` in your bundle?

**Review by weaverryan — CHANGES_REQUESTED** [review]

**Comment by yceruto** [comment]
Closing this one in favor of https://github.com/symfony/symfony-docs/pull/8687

**Pull request closed without merging**